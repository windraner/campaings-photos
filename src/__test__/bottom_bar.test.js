import React from 'react';
import BottomBar from '../components/main_container/bottom_bar/bottom_bar';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

const mock = {
  page: 1,
  perPage: 25,
  pagesCount: 10,
  stateHandler() {},
};

describe('BottomBar', () => {
  it('matches the snapshot', () => {
    const tree = shallow(<BottomBar {...mock} />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
