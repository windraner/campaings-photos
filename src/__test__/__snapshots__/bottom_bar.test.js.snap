// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomBar matches the snapshot 1`] = `
<div
  className="campaigns-photo__bottom-bar-container d-flex justify-content-between"
>
  <PerPageSelector
    stateHandler={[Function]}
    value={25}
  />
  <Pagination
    page={1}
    pagesCount={10}
    stateHandler={[Function]}
  />
</div>
`;
